### DISCUSSION TITLE
Converting from WebP to HEIC noticeably darkens image

### DISCUSSION THREAD

**Discussion by TechnoSam** [ImageMagick/ImageMagick#3238] [Help] [answered]
WebP is not currently well supported on macOS, so I'm converting a bunch of WebP files to HEIC. When I convert, the output image is noticeably darker. I installed imagemagic via homebrew and converted using the following command.

`magick convert 1.webp 1.heic`

Input: https://www.gstatic.com/webp/gallery/1.webp
Output: https://files.catbox.moe/kjsy6f.heic

Am I doing anything wrong? Am I missing dependencies or additional options needed to get a good conversion?

**Comment by snibgo** [comment, marked as answer]
What program(s) are you using to view the images?

When I convert your images to PNG files, I can't see any difference between them, and IM says the RMSE difference is only 1.2%.

Tested with IM v7.0.8-64 and Windows Photo Viewer.

**Reply by TechnoSam** [reply]
Good call, I hadn't considered that at all.

I'm using a third party app called WebPViewer from the Mac AppStore. It's certainly possible that it's artificially lightening the WebP image. I'm using the default Preview app for HEIC. Let me investigate that, I'll see if I can compare the two in a web browser.

**Reply by TechnoSam** [reply]
@snibgo Yep, that was it, my WebPViewer was actually lightening the image.

Thank you!
